collections/harmonized-landsat-sentinel.yaml

```diff
@@ -0,0 +1,526 @@
+    type:
+    title:
+  - href:
+    rel:
+    type:
+    title:
+    sentinelhub:layer_name:
+    sentinelhub:mosaicking_order:
+    sentinelhub:upsampling:
+Extent:
+  spatial:
+    bbox:
+      -
+        - -180
+        - -80
+        - 180
+        - 84
+  temporal:
+    interval:
+      -
+        - '2013-04-01T00:00:00Z'
+        - null
+CubeDimensions:
+  x:
+    type: spatial
+    axis: x
+    extent:
+      - -180
+      - 180
+    reference_system: 
+      $schema: https://proj.org/schemas/v0.4/projjson.schema.json
+      type: GeodeticCRS
+      name: AUTO 42001 (Universal Transverse Mercator)
+      datum:
+        type: GeodeticReferenceFrame
+        name: World Geodetic System 1984
+        ellipsoid:
+          name: WGS 84
+          semi_major_axis: 6378137
+          inverse_flattening: 298.257223563
+      coordinate_system:
+        subtype: ellipsoidal
+        axis:
+          - name: Geodetic latitude
+            abbreviation: Lat
+            direction: north
+            unit: degree
+          - name: Geodetic longitude
+            abbreviation: Lon
+            direction: east
+            unit: degree
+      area: World
+      bbox:
+        south_latitude: -80
+        west_longitude: -180
+        north_latitude: 84
+        east_longitude: 180
+      id:
+        authority: OGC
+        version: "1.3"
+        code: Auto42001
+  y:
+    type: spatial
+    axis: y
+    extent:
+      - 84
+      - -80
+    reference_system:
+      $schema: https://proj.org/schemas/v0.4/projjson.schema.json
+      type: GeodeticCRS
+      name: AUTO 42001 (Universal Transverse Mercator)
+      datum:
+        type: GeodeticReferenceFrame
+        name: World Geodetic System 1984
+        ellipsoid:
+          name: WGS 84
+          semi_major_axis: 6378137
+          inverse_flattening: 298.257223563
+      coordinate_system:
+        subtype: ellipsoidal
+        axis:
+          - name: Geodetic latitude
+            abbreviation: Lat
+            direction: north
+            unit: degree
+          - name: Geodetic longitude
+            abbreviation: Lon
+            direction: east
+            unit: degree
+      area: World
+      bbox:
+        south_latitude: -80
+        west_longitude: -180
+        north_latitude: 84
+        east_longitude: 180
+      id:
+        authority: OGC
+        version: "1.3"
+        code: Auto42001
+  t:
+    type: temporal
+    extent:
+      - '2013-04-01T00:00:00Z'
+      - null
+    step: P5D
+  bands:
+    type: bands
+    values:
+      - CoastalAerosol
+      - Blue
+      - Green
+      - Red
+      - RedEdge1
+      - RedEdge2
+      - RedEdge3
+      - NIR_Broad
+      - NIR_Narrow
+      - SWIR1
+      - SWIR2
+      - WaterVapor
+      - Cirrus
+      - ThermalInfrared1
+      - ThermalInfrared2
+      - QA
+      - VAA
+      - VZA
+      - SAA
+      - SZA
+      - dataMask
+sci:citation:
+Summaries:
+  eo:bands:
+    - name: CoastalAerosol
+      common_name: coastal
```

Comment:
You can still take the description from [the catalog](https://services-uswest2.sentinel-hub.com/api/v1/catalog/1.0.0/collections/hls) and add it here e.g. `"description": "Coastal Aerosol - band 1 from Landsat and Sentinel"`

Reply:
Added descriptions.